packages/design-system/src/components/OcTiles/OcTiles.vue

```diff
@@ -0,0 +1,411 @@
+      this.ghostElement.style.left = '-99999px'
+      this.ghostElement.style.top = '-99999px'
+      event.dataTransfer.setDragImage(this.ghostElement, 0, 0)
+      event.dataTransfer.dropEffect = 'move'
+      event.dataTransfer.effectAllowed = 'move'
+    },
+    dragStart(item, event) {
+      if (!this.dragDrop) return
+      this.setGhostElement(item, event)
+      this.$emit(EVENT_ITEM_DRAGGED, item)
+    },
+    dropRowEvent(selector, event) {
+      if (!this.dragDrop) return
+      const hasFilePayload = (event.dataTransfer.types || []).some((e) => e === 'Files')
+      if (hasFilePayload) return
+      this.ghostElement.remove()
+      const dropTarget = event.target
+      const dropTargetTr = dropTarget.closest('tr')
+      const dropItemId = dropTargetTr.dataset.itemId
+      this.dropRowStyling(selector, true, event)
+      this.$emit(EVENT_ITEM_DROPPED, dropItemId)
+    },
+    dropRowStyling(selector, leaving, event) {
+      const hasFilePayload = (event.dataTransfer.types || []).some((e) => e === 'Files')
+      if (hasFilePayload) return
+      if (event.currentTarget?.contains(event.relatedTarget)) {
+        return
+      }
+
+      const classList = document.getElementsByClassName(`oc-tbody-tr-${selector}`)[0].classList
+      const className = 'highlightedDropTarget'
+      leaving ? classList.remove(className) : classList.add(className)
+    },
+    isFieldTypeSlot(field) {
+      return field.type === 'slot'
+    },
+    isFieldTypeCallback(field) {
+      return ['callback', 'function'].indexOf(field.type) >= 0
+    },
+    extractFieldTitle(field) {
+      if (Object.prototype.hasOwnProperty.call(field, 'title')) {
+        return field.title
+      }
+      return field.name
+    },
+    isHighlighted(item) {
+      if (!this.highlighted) {
+        return false
+      }
+
+      if (Array.isArray(this.highlighted)) {
+        return this.highlighted.indexOf(item[this.idKey]) > -1
+      }
+
+      return this.highlighted === item[this.idKey]
+    },
+    isDisabled(item) {
+      if (!this.disabled) {
+        return false
+      }
+
+      if (Array.isArray(this.disabled)) {
+        return this.disabled.indexOf(item[this.idKey]) > -1
+      }
+
+      return this.disabled === item[this.idKey]
+    },
+
+    cellKey(field, index, item) {
+      const prefix = [item[this.idKey], index + 1].filter(Boolean)
+
+      if (this.isFieldTypeSlot(field)) {
+        return [...prefix, field.name].join('-')
+      }
+
+      if (this.isFieldTypeCallback(field)) {
+        return [...prefix, field.callback(item[field.name])].join('-')
+      }
+
+      return [...prefix, item[field.name]].join('-')
+    },
+    getSortLabel(name) {
+      return this.$gettextInterpolate(this.$gettext('Sort by: %{ name }'), { name })
+    }
+  }
+}
+</script>
+<style lang="scss" scoped>
+@import '../../styles/styles';
+
+ul.tiles-list {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: row;
+  gap: 15px;
+  justify-content: flex-start;
```

Comment:
```suggestion
  justify-content: space-around;
```
is much more pleasant for the eye, but any time the last row is incomplete (i.e. not the same number of tiles as in the other rows) the remaining tiles are centered, which doesn't look good... maybe we need some magic dummy tiles.

Reply:
The horizontal spacing is nice, but then the vertical spacing doesn't match - with this change alone